### PR TITLE
Feature/validate role options

### PR DESCRIPTION
**Pull request by alexeyklyukin**
Only accept CREATE ROLE flags that doesn't have any params (i.e.
not ADMIN or CONNECTION LIMIT). Check that both flag and NOflag
are not used at the same time.

### CODE REVIEW

File: pkg/cluster/util.go [hunk 1]
```diff
@@ -21,23 +21,47 @@ func isValidUsername(username string) bool {
...
+			return true
+		}
+	}
+	return false
+}
+
+func invertFlag(flag string) string {
+	if flag[:2] == "NO" {
+		return flag[2:]
+	} else {
```
Comment (ikitiki) [inline pkg/cluster/util.go]: no need for ``else``

File: pkg/cluster/util.go [hunk 2]
```diff
@@ -21,23 +21,47 @@ func isValidUsername(username string) bool {
...
+		return "NO" + flag
+	}
+}
+
 func normalizeUserFlags(userFlags []string) (flags []string, err error) {
```
Comment (ikitiki) [inline pkg/cluster/util.go]: I think in this case there is no strong need for the named return values.
Besides, code like ``return nil, fmt.Errorf("bla")`` as for me looks more natural than 
```
err = fmt.Errorf("bla")
return
```

**Comment by ikitiki** [comment]
👍

**Comment by alexeyklyukin** [comment]
👍

**Pull request merged**